sql/malsync/malsync.go

```diff
@@ -8,6 +8,11 @@ import (
 )
 
 func GetSyncState(db sql.Executor) (time.Time, error) {
+	timestamp, _, err := getSyncState(db)
+	return timestamp, err
+}
+
+func getSyncState(db sql.Executor) (time.Time, bool, error) {
```

Comment:
It's to see if we need to create a new row (the table has just 1 row) or use an existing one
It would probably better to use `INSERT ... ON CONFLICT` ("upsert") but that would require another migration to add a primary key to the table
So I'll make return values named I think

Reply:
There are two callers of `getSyncState`, one only is interested in the timestamp and one only is interested in the boolean. Why not make `getSyncState` two functions then? 🙂

Reply:
> It would probably better to use INSERT ... ON CONFLICT ("upsert") but that would require another migration to add a primary key to the table

I'm actually for the suggested migration. Why not enforce on DB level that there can only be one timestamp in the DB instead of doing multiple checks in code to ensure there is only one? The migration should also be quick to apply or not? 🙂

Reply:
Added a migration and switched to using `INSERT ... ON CONFLICT`